concourse/steps/replicate_secrets.py

```diff
@@ -153,13 +154,48 @@ def replicate_secrets(
     logger.info(f'deployed encrypted secret for team: {team_name}')
 
 
+def _issue_type_for_policy_violation(
+    policy_violation: cmm.CfgStatusEvaluationAspects,
```

Comment:
I seem to have missed this in previous PRs. please choose a more adequate name for this enum. do we even need this enum? just use the type-labels. then you even save some code (this function, ..)

as for the name: those are compliance _violations_. An "evaluation aspect" could be the inspection of whether or not a certain criterion is met. This is not identical to the _result_ of such a check. Please, as mentioned before, take care to choose names that match the semantics. Use simple terms, do not add verbal boilerplate.

Reply:
I agree, the naming is not matching the semantics. Let's handle renaming `CfgStatusEvaluationAspects` -> `CfgStatusComplianceViolations` in a separate PR.